bin/run_e2e_tests

```diff
@@ -17,17 +17,33 @@ print_error () {
 }
 
 setup () {
+    ${dockerComposeCmd} up -d db # start database first - it takes about 10 sec untils db startsup, and otherwise db migration fails
+    if [ ! $? -eq 0 ]; then
+        print_error "Db startup failed"
+        cleanup
+        exit 1
+    fi
+
     ${dockerComposeCmd} build
     if [ ! $? -eq 0 ]; then
         print_error "Building docker images failed"
         exit 1
     fi
 
+    ${dockerComposeCmd} run --entrypoint bin/db-upgrade discovery
+    if [ ! $? -eq 0 ]; then
+        print_error "Db migration failed"
+        cleanup
+        exit 1
```

Comment:
These 3 lines repeats on each step - we can make a function `fatal(error_message)` to DRY this.

Reply:
It's bash - next time